### PR TITLE
LPS-131640 Inline context HashMap creation in JSPs

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

File: modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/export_process_action.jsp [hunk 1]
```diff
@@ -46,7 +42,11 @@ BackgroundTask backgroundTask = (BackgroundTask)row.getObject();
 		%>
 
 		<liferay-ui:icon
-			data="<%= data %>"
+			data='<%=
+				HashMapBuilder.<String, Object>put(
+					"senna-off", "true"
+				).build()
+			%>'
```
Comment (drewbrokke) [inline modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/export_process_action.jsp:49]: Does `Collections.singletonMap` compile if used here? If so let's use that instead.

**Review by drewbrokke — COMMENTED** [review]
Looks good @erickmonteiroo, just the one question. If we can't use it then this is good to go.

File: modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/export_process_action.jsp (hunk 1 above)
Comment (erickmonteiroo) [reply modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/export_process_action.jsp:49]: It does! Resending.

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Review by drewbrokke — APPROVED** [review]
LGTM 👍

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**